### DISCUSSION TITLE
A tour of elementary's input stack

### DISCUSSION THREAD

**Discussion by JoseExposito** [elementary/edw#36] [Withdrawn or Not Accepted]
Speaker: @JoseExposito

A quick overview of how the current input stack looks like, the problems we found on X11 to implement multi-touch support at the OS level, the solutions we came with and the bright future on Wayland.

If Odin is not released by the talk, I'll make the talk shorter to fit a demo in.

**Comment by danirabbit** [comment]
For this conference, we definitely want to keep the focus on app developers. To me, this kind of sounds like information that is mostly for either platform developers or curious users maybe. Can you elaborate on how this talk can help developers who are writing apps?

**Reply by JoseExposito** [reply]
This was going to be an "under the hood" or "how it works" talk. Save the slot for someone else, this is not useful at all for app developers :smile:
